Cargo.toml

```diff
@@ -25,7 +25,7 @@ curl = { version = "0.4.44", features = ["http2"] }
 curl-sys = "0.4.58"
 env_logger = "0.9.0"
 pretty_env_logger = { version = "0.4", optional = true }
-anyhow = "1.0"
+anyhow = "1.0.47"
```

Comment:
Thanks!

I feel like we can set at least at `1.0.65`, since Cargo.lock under rust-lang/rust [already got that version](https://github.com/rust-lang/rust/blob/2afca78a0b03db144c5d8b9f8868feebfe096309/Cargo.lock#L106).

Reply:
Yeah, I wasn't sure if Cargo wants a maximal or minimal bump. 1.0 seems like the minimal policy.